samples/matter/lock/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ &dppic130 {
```

Comment:
Can we have some comments to this channel configuration? I don't understand anything what is going on here :)

Reply:
Those configurations will be moved to the Zephyr common directory, so we don't need to take care of it. For now, it is a workaround and this file will be removed once it is supported in sdk-zephyr. I will add a note there that it is a workaround.